skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -19,12 +19,16 @@ import kotlin.math.min
 class SkikoUIView : UIView, UIKeyInputProtocol, UITextInputProtocol,
     org.jetbrains.skiko.objc.UIViewExtensionProtocol {
     @OverrideInit
-    constructor(frame: CValue<CGRect>) : super(frame)
+    constructor(frame: CValue<CGRect>) {
+        error("use SkikoUIView constructor with skiaLayer")
```

Comment:
Can we add some annotation to throw an error in compile time?

Reply:
Added Deprecated annotation with error level. It will fail if accidentally used on compilation.